install/Makefile

```diff
@@ -74,7 +74,7 @@ APACHE_PROGRAM           = /usr/sbin/apache2
 
 # For testing proxying of an external domain, this represents the domain we
 # are proxying from.
-PAGESPEED_TEST_HOST ?= modpagespeed.com
+PAGESPEED_TEST_HOST ?= selfsigned.modpagespeed.com
```

Comment:
@morlovich re: "Maybe the second one can be renamed, and then some of the places it's set can be removed?" 
I think that makes sense, but I would prefer to do that in separate follow up (merging this should stop CI failing). Would that work for you?